src/views/partials/go-back.ejs

```diff
@@ -0,0 +1,11 @@
+<div class="card-button-container fr-mt-2w">
```

Comment:
put this directly in verify-email.ejs. We will do the uniformization in another PR.